hummingbot/connector/exchange/bittrex/bittrex_exchange.py

```diff
@@ -0,0 +1,403 @@
+                           time_provider=self._time_synchronizer)
+
+    @property
+    def rate_limits_rules(self):
+        return CONSTANTS.RATE_LIMITS
+
+    @property
+    def domain(self):
+        return CONSTANTS.DEFAULT_DOMAIN
+
+    @property
+    def client_order_id_max_length(self):
+        return CONSTANTS.MAX_CLIENT_ID_LENGTH
+
+    @property
+    def client_order_id_prefix(self):
+        return ""
+
+    @property
+    def trading_rules_request_path(self):
+        return CONSTANTS.EXCHANGE_INFO_PATH_URL
+
+    @property
+    def trading_pairs_request_path(self):
+        return CONSTANTS.EXCHANGE_INFO_PATH_URL
+
+    @property
+    def check_network_request_path(self):
+        return CONSTANTS.SERVER_TIME_URL
+
+    @property
+    def trading_pairs(self):
+        return self._trading_pairs
+
+    @property
+    def is_cancel_request_in_exchange_synchronous(self) -> bool:
+        return True
+
+    @property
+    def is_trading_required(self) -> bool:
+        return self._trading_required
+
+    def supported_order_types(self):
+        return [OrderType.LIMIT, OrderType.LIMIT_MAKER]
+
+    def _is_request_exception_related_to_time_synchronizer(self, request_exception: Exception):
+        return False
+
+    def _create_web_assistants_factory(self) -> WebAssistantsFactory:
+        return web_utils.build_api_factory(
+            time_synchronizer=self._time_synchronizer,
+            auth=self._auth)
+
+    def _create_order_book_data_source(self) -> OrderBookTrackerDataSource:
+        return BittrexAPIOrderBookDataSource(
+            trading_pairs=self._trading_pairs,
+            connector=self,
+            api_factory=self._web_assistants_factory)
+
+    def _create_user_stream_data_source(self) -> UserStreamTrackerDataSource:
+        return BittrexAPIUserStreamDataSource(auth=self._auth)
+
+    async def _place_order(self,
+                           order_id: str,
+                           trading_pair: str,
+                           amount: Decimal,
+                           trade_type: TradeType,
+                           order_type: OrderType,
+                           price: Decimal,
+                           **kwargs
+                           ) -> Tuple[str, float]:
+
+        path_url = CONSTANTS.ORDER_CREATION_URL
+        body = {
+            "marketSymbol": await self.exchange_symbol_associated_to_pair(trading_pair=trading_pair),
+            "direction": trade_type.name,
+            "type": "LIMIT",
+            "quantity": f"{amount:f}",
+            "limit": f"{price:f}",
+        }
+        if order_type is OrderType.LIMIT:
+            body.update({
+                "timeInForce": "GOOD_TIL_CANCELLED"
+            })
+        elif order_type is OrderType.LIMIT_MAKER:
+            body.update({
+                "timeInForce": "POST_ONLY_GOOD_TIL_CANCELLED"
+            })
+        body.update({
+            "clientOrderId": order_id
+        })
```

Comment:
Please move the configuration of the clientOrderId in the dictionary to the creation of the body dictionary (line 137)